src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1185,6 +1195,31 @@ protected CompilerOptions createOptions() {
 
     options.setConformanceConfigs(loadConformanceConfigs(flags.conformanceConfigs));
 
+    if ( ! flags.instrumentationFile.isEmpty()) {
+        String instrumentationPb;
+        Instrumentation.Builder builder = Instrumentation.newBuilder();
+        try {
```

Comment:
Can you convert this try into a try-with-resources?
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html